.github/CONTRIBUTNG.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+
+Please read the PAVICS [Developer Documentation](https://pavics-sdi.readthedocs.io/en/latest/dev/index.html) to get started.
+
+## Policy
+Since PAVICS is used in production by multiple organizations, this deployment repository also has a policy regarding contributions. 
+
+### Policy objectives:
+1. Keep production stable between upgrades
+2. Encourage PR to be merged more quickly
+3. Achieve the two previous objectives 
+      * by weighing down everyone's workflow as much as possible
+      * by having an approach that scales well (increase of production site and at the same time increase of the number of PR)
+
+### Policy rules:
+1. The repository has a main branch, "master", open to the community where contributions, "PR", are welcome. This master branch must not have owners and therefore no organization can block contributions to it.
+2. Contributions should be backward-compatible as much as possible or feature a toggle so that organizations can activate the new feature when ready without being disturbed in the meantime.
```

Comment:
```suggestion
2. Contributions should be backward-compatible whenever possible, or feature a toggle switch so that organizations can activate the new feature on their own schedule.
```